.github/workflows/upload-contrib-stats.yml

```diff
@@ -1,14 +1,19 @@
 name: Upload contribution stats
 
 on:
+  pull_request:
   schedule:
     # Choose a random time near midnight PST because it may be delayed if there are high loads
     - cron:  37 7 * * *
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}-${{ github.event_name == 'workflow_dispatch' }}
```

Comment:
I think you mean `-${{ github.event_name == 'schedule' }}` here as this workflow only run on schedule (after the `pull_request` part is removed per the above comment

Reply:
Gotcha, I just copied what was in the error message haha